### DISCUSSION TITLE
Blis shared libs

### DISCUSSION THREAD

**Discussion by dimidloc** [explosion/spaCy#11041] [Help: Installation] [answered]
Hello, I'm trying to cut down the size of spacy to serve it as a web service. When installing spacy via pip, one of the dependencies is `blis`. It installs two `so` files, and both of them weigh 15M:

cy.cpython-38-x86_64-linux-gnu.so
py.cpython-38-x86_64-linux-gnu.so
 
I was wondering if both of them are actually needed for spacy?

**Comment by adrianeboyd** [comment, marked as answer]
These are definitely needed for spacy.
